es-reindexers/systemd/perfscale-es-ols-load-test-results-ocp-qe.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
-Description=Perfscale ES ingress-performance dev sync job
-Wants=perfscale-es-ingress-performance-dev.timer
+Description=Perfscale ES OLS load test results ocp qe sync job
+Wants=perfscale-es-ols-load-test-results-ocp-qe.timer
 
 [Service]
 Type=simple
 User=root
 WorkingDirectory=/root
 # This driver script can be found in the jump host: snowflake.rdu2.scalelab.redhat.com. Hiding it as it has contains secrets.
-ExecStart=/bin/bash perfscale-es-ingress-performance-dev.sh
+ExecStart=/bin/bash perfscale-es-ols-load-test-results-ocp-qe.sh
```

Comment:
I see that you are requiring this script `perfscale-es-ols-load-test-results-ocp-qe.sh`, but I don't see it in the repo, or your branch.

Reply:
All these scripts have some secrets exposed as ENVs, so we do not keep them here in the repo. They are present in the jump host and are being used as drivers for the systemd timers.